packages/web-frontend/src/sagas.js

```diff
@@ -986,9 +987,19 @@ function* watchGoHomeAndResetToProjectSplash() {
   yield takeLatest(GO_HOME, resetToProjectSplash);
 }
 
+function* resetToDefaultProject() {
+  // TODO: make sure that this function is considered in the GO_HOME PR.
```

Comment:
Yeah let's make sure that the defaults work properly before this epic is ready to be tested!